app/build.gradle

```diff
@@ -130,7 +130,7 @@ dependencies {
 	implementation "com.google.dagger:dagger:$daggerVersion"
 	kapt "com.google.dagger:dagger-compiler:$daggerVersion"
 
-	implementation('com.gitlab.opentransitmap:public-transport-enabler:9c909718') {
```

Comment:
Did you test this? Looks like you didn't update the dependency pinning. See https://github.com/grote/Transportr/pull/479#issuecomment-407740999 for more details!

Reply:
@grote You removed verification in https://github.com/grote/Transportr/commit/4bd5924b65bafc4e18f582d6e6ae6fd81ee8d29d#diff-39e7d8c00954e920b98e7636f0ac30b2

Reply:
Shall I add it back?